src/services/symbolication.rs

```diff
@@ -1653,7 +1657,26 @@ impl SymbolicationActor {
                     // Stackwalk the minidump.
                     let cfi = cfi_caches.load_cfi();
                     let minidump = ByteView::from_slice(&minidump);
-                    let process_state = ProcessState::from_minidump(&minidump, Some(&cfi))?;
+                    let timer_old = Instant::now();
+                    let process_state =
+                        ProcessState::from_minidump_breakpad(&minidump, Some(&cfi))?;
+                    let timer_old = timer_old.elapsed();
+
+                    // Run and time the new stackwalking method if compare_stackwalking_methods was passed
+                    let process_state_and_timer_new = if compare_stackwalking_methods {
+                        let timer_new = Instant::now();
+                        match ProcessState::from_minidump(&minidump, Some(&cfi)) {
+                            Ok(state) => Some((state, timer_new.elapsed())),
+                            Err(_) => {
+                                // If the old method succeeded and the new one failed, report this fact
+                                metric!(counter("minidump.stackwalk.new_method.errors") += 1);
+                                None
+                            }
```

Comment:
You might also want to capture this error in sentry?

Reply:
How do I do this?